codespell_lib/data/dictionary.txt

```diff
@@ -21154,6 +21158,9 @@ odly->oddly
 ody->body
 oen->one
 oerflow->overflow
+oerflowed->overflowed
```

Comment:
This was old English, and is used in a few poems so perhaps rare? Maybe for all of those versions?

Reply:
I'm fine with that, pushed a new version of the PR with this suggestion applied.